### PR TITLE
tweaking M2 germline filter

### PR DESCRIPTION
**Pull request by davidbenjamin**
@LeeTL1220 Can you review?  This fixes your bug.  I'm not putting in a regression test yet because i the upcoming filtering refactoring it will become much easier to unit test for bugs like this.

@madduran Yours too.

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java [hunk 1]
```diff
@@ -265,17 +263,22 @@ private void applyGermlineVariantFilter(final M2FiltersArgumentCollection MTFAC,
 
             final int refCount = alleleCounts[0];
 
+            final int totalCount = (int) MathUtils.sum(alleleCounts);
```
Comment (LeeTL1220) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java]: I am assuming that there is no `int MathUtils.sum(int[])`...

**Comment by davidbenjamin** [comment]
These are failing with:

```
Error: (converted from warning) unable to access index for repository http://lib.stat.cmu.edu/R/CRAN/src/contrib
Execution halted
The command "if [[ $TEST_DOCKER != true ]]; then sudo mkdir -p /usr/local/lib/R/; sudo mkdir -p site-library; sudo ln -sFv ~/site-library /usr/local/lib/R/site-library; sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9; sudo add-apt-repository "deb http://cran.rstudio.com/bin/linux/ubuntu trusty/"; sudo apt-get update; sudo apt-get install -y --force-yes r-base-dev=3.1.3-1trusty; sudo apt-get install -y --force-yes r-base-core=3.1.3-1trusty; sudo Rscript scripts/docker/gatkbase/install_R_packages.R; fi;" failed and exited with 1 during
```
which has nothing to do with the PR.

**Comment by LeeTL1220** [comment]
@davidbenjamin I think @jonn-smith (or someone else on the engine team) knows how to get around the R issue.

**Comment by LeeTL1220** [comment]
@davidbenjamin (Maybe just rebase to master)

**Pull request merged**